Mathlib/Combinatorics/SimpleGraph/Metric.lean

```diff
@@ -128,6 +128,15 @@ theorem edist_eq_one_iff_adj : G.edist u v = 1 ↔ G.Adj u v := by
     exact w.adj_of_length_eq_one <| Nat.cast_eq_one.mp <| h ▸ hw
   · exact le_antisymm (edist_le h.toWalk) (ENat.one_le_iff_pos.mpr <| edist_pos_of_ne h.ne)
 
+lemma edist_bot [DecidableEq V] :
+    ∀ u v, (⊥ : SimpleGraph V).edist u v = (if u = v then 0 else ⊤) := by
```

Comment:
Why not?
```lean
lemma edist_bot [DecidableEq V] (u v : V) : (⊥ : SimpleGraph V).edist u v = (if u = v then 0 else ⊤)
```
(And the same in the others)

Reply:
Good point, thank you! ^^